scripts/install/install_macos.sh

```diff
@@ -594,6 +591,13 @@ create_supervisor_config() {
 
   info "Creating supervisor config..."
 
+  if [ -z "$OPAMP_ENDPOINT" ]; then
+    OPAMP_ENDPOINT="ws://localhost:3000/v1/opamp"
+    increase_indent
+    info "No OpAMP endpoint specified, starting agent using 'ws://localhost:3001/v1/opamp' as endpoint."
```

Comment:
Line 595 has port `3000`, not `3001`. Is this a problem? Looks like it deployed correctly for me, with 3001.

Reply:
Yea good catch, that should be `3001`, I was just trying to use the default for a local BP server as the default OpAmp endpoint.